### DISCUSSION TITLE
Is there a better way to bind react and prose-mirror?

### DISCUSSION THREAD

**Discussion by 4-1-1** [outline/rich-markdown-editor#468] [Ideas] [answered]
I want to try extend it with web components.  
I don't know if it able to work。if it works, it will be easier to do something , like use other react package

**Comment by tommoor** [comment, marked as answer]
See the `Embed` node for an example: https://github.com/outline/rich-markdown-editor/blob/main/src/nodes/Embed.tsx

In general it's best avoided where possible – lots of complications.
